### DISCUSSION TITLE
What is the status of TextField?

### DISCUSSION THREAD

**Discussion by tsemerad** [wix/react-native-ui-lib#1774] [Q&A] [answered]
I want to use TextField in my app. I see it on the documentation website, but it only shows TextField in the Incubator section. However, there is also an older TextField component exported from the library.

Is it recommended to use the new Incubator TextField? If not, why are the docs for the older TextField not on the docs website? If there's anywhere I can contribute to help resolve this confusion for future users, let me know :)

**Comment by ethanshar** [comment, marked as answer]
Hi @tsemerad 
That's a great question!

The `Incubator.TextField` is the new and improved implementation of our TextField component. 
It is under Incubator because it was in experimental state, but it's pretty much in stable state now and can be used. 

In our next major version (v7) we will replace the old TextField with the new one, and make the Incubator.TextField official.
